### PR TITLE
Add content for eBPF documentation

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
We are working to improve our documentation and create a more meaningful dashboard information, this PR is the first PR addressing this task.

The texts inserted here are actually in our dashboard, but they will be removed in next PR.
##### Test Plan

1. tests are not necessary, because we are only working with documentation.

##### Additional Information

The next PR will address content on dashboard related to these topics.

<details> <summary>For users</summary>
This PR is updating our documentation to give more information for users when they are monitoring file systems. We will have a PR that will create links for this documentation.
</details>

### CODE REVIEW

File: collectors/ebpf.plugin/README.md [hunk 1]
```diff
@@ -410,6 +420,11 @@ is accessed, but also in possible errors, so we need to attach a `kretprobe`. Fo
 - [`d_lookup`](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/dcache.c?id=052b398a43a7de8c68c13e7fa05d6b3d16ce6801#n2223):
   Called when the desired file is not inside the directory cache.
 
+##### Directory Cache Interpretation
+When directory cache is showing 100% means that every file that was accessed was present in the directory cache.
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:424]:
```suggestion
When directory cache is showing 100%, that means that every file that was accessed was present in the directory cache.
```

File: collectors/ebpf.plugin/README.md [hunk 2]
```diff
@@ -410,6 +420,11 @@ is accessed, but also in possible errors, so we need to attach a `kretprobe`. Fo
...
+If files are not present in the directory cache 1) they are not present in the file system, 2) the files were not
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:425]: Is 1 and 2 an "and" or "or" situation?

File: collectors/ebpf.plugin/README.md [hunk 3]
```diff
@@ -244,19 +244,29 @@ To collect data related to Linux multi-device (MD) flushing, the following kprob
...
 ### Filesystem
 
-This group has charts demonstrating how applications interact with the Linux
-kernel to open and close file descriptors. It also brings latency charts for
-several different filesystems.
+This group has charts demonstrating how applications interact with the Linux kernel to open and close file descriptors.
+It also brings latency charts for several different filesystems.
+
+#### Latency Algorithm
+We calculate the difference between the calling and return times, this spans disk I/O, file system operations (lock, I/O),
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:268]:
```suggestion
We calculate the difference between the calling and return times, spanning disk I/O, file system operations (lock, I/O),
```

File: collectors/ebpf.plugin/README.md [hunk 4]
```diff
@@ -244,19 +244,29 @@ To collect data related to Linux multi-device (MD) flushing, the following kprob
...
+This will create the chart `disk_latency_io` for each disk on the host. The following tracepoints are used:
 
 - [`block/block_rq_issue`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_issue):
   IO request operation to a device drive.
 - [`block/block_rq_complete`](https://www.kernel.org/doc/html/latest/core-api/tracepoint.html#c.trace_block_rq_complete):
   IO operation completed by device.
 
+Disk Latency is the single most important metric to focus on when it comes to storage performance, under most circumstances.
+For hard drives, an average latency somewhere between 10 to 20 ms can be considered acceptable. For SSD (Solid State Drives),
+depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:259]:
```suggestion
depending on the workload it should never reach higher than 1-3 ms. In most cases, workloads will experience less than
```

I'm going to suggest we combine this sentence and the next one to say "In most cases, workloads experience less than 1 ms latency numbers, but workloads should never reach higher than 3 ms."

**Review by DShreve2 — CHANGES_REQUESTED** [review]

File: collectors/ebpf.plugin/README.md [hunk 5]
```diff
@@ -410,6 +420,11 @@ is accessed, but also in possible errors, so we need to attach a `kretprobe`. Fo
 - [`d_lookup`](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/dcache.c?id=052b398a43a7de8c68c13e7fa05d6b3d16ce6801#n2223):
   Called when the desired file is not inside the directory cache.
 
+##### Directory Cache Interpretation
+When directory cache is showing 100% means, that means that every file that was accessed was present in the directory cache.
+If files are not present in the directory cache they are not present in the file system, or the files were not
```
Comment (DShreve2) [inline collectors/ebpf.plugin/README.md:425]:
```suggestion
If files are not present in the directory cache, they are either not present in the file system or the files were not
```

**Review by DShreve2 — DISMISSED** [review]

File: collectors/ebpf.plugin/README.md [hunk 6]
```diff
@@ -410,6 +420,11 @@ is accessed, but also in possible errors, so we need to attach a `kretprobe`. Fo
 - [`d_lookup`](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/fs/dcache.c?id=052b398a43a7de8c68c13e7fa05d6b3d16ce6801#n2223):
   Called when the desired file is not inside the directory cache.
 
+##### Directory Cache Interpretation
+When directory cache is showing 100% means, that means that every file that was accessed was present in the directory cache.
```
Comment (kickoke) [inline collectors/ebpf.plugin/README.md:424]:
```suggestion
When directory cache is showing 100% that means that every accessed file was present in the directory cache.
```

**Review by kickoke — APPROVED** [review]
LGTM. Thank you 🏅

**Review by DShreve2 — APPROVED** [review]

**Pull request merged**